### PR TITLE
fix: Add last_active_at to Tenant

### PR DESCRIPTION
**Pull request by filipecabaco**
## What kind of change does this PR introduce?

Adds last_active_at to Tenant so we can use it in the Janitor module which will handle tenant cleanup of partitions.

For now we will only clean up partitions of the users that have been online during the last 5 days

### CODE REVIEW

File: lib/realtime/tenants.ex [hunk 1]
```diff
@@ -287,11 +292,62 @@ defmodule Realtime.Tenants do
...
+  An active tenant is a tenant that has `last_active_at` higher than the given days.
+
+  Defaults to 5 days.
+  """
+  def stream_active_tenants(days \\ 5) do
+    region = Application.get_env(:realtime, :region)
+    regions = Nodes.region_to_tenant_regions(region)
+    region_nodes = Nodes.region_nodes(region)
+
+    from(t in Tenant,
```
Comment (abc3) [inline lib/realtime/tenants.ex:318]: how should we handle new customers who have worked for more than five days but have not yet been terminated?

Comment (filipecabaco) [reply lib/realtime/tenants.ex:320]: updated to be in handle_continue

File: lib/realtime/tenants/janitor.ex [hunk 2]
```diff
@@ -96,7 +84,8 @@ defmodule Realtime.Tenants.Janitor do
 
   def handle_info({:DOWN, ref, _, _, :normal}, state) do
```
Comment (chasers) [inline lib/realtime/tenants/janitor.ex:85]: You're handling the failed case but doesn't look like the successful case. 

Seems like your state would grow unbounded with tasks over time. 

The async_nolink docs:

https://hexdocs.pm/elixir/Task.Supervisor.html#async_nolink/3

```elixir
  # The task completed successfully
  def handle_info({ref, answer}, %{ref: ref} = state) do
    # We don't care about the DOWN message now, so let's demonitor and flush it
    Process.demonitor(ref, [:flush])
    # Do something with the result and then return
    {:noreply, %{state | ref: nil}}
  end
  ```

**Review by chasers — CHANGES_REQUESTED** [review]

File: lib/realtime/tenants/janitor.ex (hunk 2 above)
Comment (chasers) [reply lib/realtime/tenants/janitor.ex:85]: Oh wait... I see it now.

Comment (chasers) [reply lib/realtime/tenants/janitor.ex:85]: I guess because you don't care about the response from the task you're handling the normal down message.

**Review by chasers — APPROVED** [review]

**Comment by filipecabaco** [comment]
https://github.com/supabase/realtime/pull/1201

**Pull request closed without merging**